templates/display.html

```diff
@@ -2,9 +2,9 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <title>Microsoft Identity Python Web App: API</title>
 </head>
 <body>
-    <a href="javascript:window.history.go(-1)">Back</a> <!-- Displayed on top of a potentially large JSON response, so it will remain visible -->
```

Comment:
Removed back button since you can just use browser back button?

Reply:
I would not recommend doing that.

You probably know that, in many other everyday websites, depending on their actual implementation, the browser back button does not always work well. It sometimes gives an unfriendly "the page needs you to resubmit the form" message. So, I am a strong believer that a good website should provide its own back button in its UI.

That is why this button was introduced here. The fact that it happens to utilize the browser's back mechanism is an implementation detail which does not negate the aforementioned principle.

Reply:
Okay, I'm not very opinionated about it, I put it back.